Resources/Prototypes/Entities/Clothing/Masks/cig.yml

```diff
@@ -0,0 +1,16 @@
+﻿- type: entity
+  parent: MasksBase
+  id: Cigarette
+  name: cigarette
+  description: "If you want to get cancer, might as well do it in style."
+  components:
```

Comment:
Add an item component here, and give it the correct unlit prefix.

Reply:
Could just inherit from BaseItem instead?